### PR TITLE
Cleanup the node instance table on startup

### PR DESCRIPTION
**Pull request by stelfrag**
##### Summary
If the host has been deleted from the database, cleanup the related node instance entries from the database

##### Test Plan
- Should be self explanatory

### CODE REVIEW

**Review by MrZammler — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

File: database/sqlite/sqlite_functions.c [hunk 1]
```diff
@@ -62,6 +62,7 @@ const char *database_cleanup[] = {
     "delete from chart where chart_id not in (select chart_id from dimension);",
```
Comment (thiagoftsm) [inline database/sqlite/sqlite_functions.c:62]: I already approved the PR, because the goal was reached according the tests I ran. A small suggestion that can be address in another PR, this pointer has SQ keywords in lowercase and the last is in uppercase, considering that we have other pointers with SQL keyword  in uppercase, my suggestion is also convert the lowercase here to uppercase.
